src/openms/source/FILTERING/CALIBRATION/TOFCalibration.cpp

```diff
@@ -195,7 +195,7 @@ namespace OpenMS
   void TOFCalibration::getMonoisotopicPeaks_(PeakMap & calib_peaks, std::vector<std::vector<unsigned int> > & monoiso_peaks)
   {
 
-    PeakMap::iterator spec_iter;
+    [[maybe_unused]] PeakMap::iterator spec_iter;
```

Comment:
did you check if the variable is used? if not you can remove it

Reply:
spec_iter seems to be used in a lot of instances

iterator it wasn't used, so I removed that

Reply:
the one below is not the same variable!

Reply:
My bad! I should have watched more closely

I'll make changes